src/libs/ReportActionsUtils.ts

```diff
@@ -313,11 +313,11 @@ function shouldIgnoreGap(currentReportAction: ReportAction | undefined, nextRepo
  * transaction thread report in order to correctly display reportActions from both reports in the one-transaction report view.
  */
 function getCombinedReportActions(reportActions: ReportAction[], transactionThreadReportActions: ReportAction[], reportID?: string): ReportAction[] {
-    if (isEmptyObject(transactionThreadReportActions)) {
+    if (isEmptyObject(transactionThreadReportActions) || reportActions.some((action) => isSentMoneyReportAction(action))) {
         return reportActions;
```

Comment:
Can you add more context about this change? Also, `reportActions` are not sorted which make last message in pay report is not correct

Reply:
In this case we well return all report action of the expense report which have already been sorted in the report screen.

Reply:
What does `reportActions.some((action) => isSentMoneyReportAction(action))` mean?

Reply:
Also, we need to fix this
> `reportActions` are not sorted which make last message in pay report in LHN is not correct

Reply:
No, this only return true when we open the report preview of send money request.

Reply:
I think it will be more clean if we define it in a const and add some coments why we return `reportActions` in SentMoneyReport
```javascript
const isSentMoneyReport = reportActions.some((action) => isSentMoneyReportAction(action))
```

Reply:
@ahmedGaber93 I updated.

Reply:
@nkdengineer We still need to fix this https://github.com/Expensify/App/pull/43742#discussion_r1648684528
Steps to reproduce
1. set Priority mode to most recent
2. pay someone
3. open pay report
4. send message
the message you sent it in step 3 not visible in LHN as last message you sent

Reply:
@ahmedGaber93 I fixed this case.